examples/fields.go

```diff
@@ -51,16 +50,16 @@ func main() {
 
 	// set attribute fields
 	var volAttrList = []string{
-		*nimbleos.VolumeFields.ID,
-		*nimbleos.VolumeFields.Name,
-		*nimbleos.VolumeFields.PerfpolicyName,
+		nimbleos.VolumeFields.ID,
+		nimbleos.VolumeFields.Name,
+		nimbleos.VolumeFields.PerfpolicyName,
 	}
 	// apply attributes
 	filter.WithFields(volAttrList)
 
 	// create a filter
 	sf := &param.SearchFilter{
-		FieldName: nimbleos.VolumeFields.Name,
+		FieldName: param.NewString(nimbleos.VolumeFields.Name),
```

Comment:
Hmmm...  I didn't see this coming.  Maybe we could use &nimbleos.VolumeFields.Name without the param.NewString() function.  But we might be better off using the SearchFilter Init() function.

I see the same thing shows up in a bunch of the services.

I'm tempted to dig into why FieldName is a pointer, but I'll resist....

Reply:
I'm not sure what you mean.. Should these be
```
sf := &param.SearchFilter{
	    FieldName: &nimbleos.VolumeFields.Name,
            ...
        }
```
?

Reply:
Right, that should work.  I noticed there is an Init() function for SearchFilter which takes a string rather than a string pointer.  I'm not sure exactly how it is intended to be used.  It might not be worth considering here.